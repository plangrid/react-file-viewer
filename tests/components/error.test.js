// Copyright (c) 2017 PlanGrid, Inc.

import React from 'react';
import renderer from 'react-test-renderer';

import Error from 'components/error';

describe('Error component', () => {
  it('matches stored snapshot', () => {
    const tree = renderer.create(
      <Error />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
