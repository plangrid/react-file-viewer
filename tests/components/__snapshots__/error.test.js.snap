// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component matches stored snapshot 1`] = `
<div
  className="error-message"
>
  <p
    className="alert"
  >
    Unable to preview file
  </p>
</div>
`;
